molecule/default/tests/test_default.py

```diff
@@ -20,6 +20,8 @@ def test_packages(host):
         # Debian Bullseye is not yet supported by the official Docker
         # package repo
         assert host.package("docker.io").is_installed
+    elif distribution == "fedora":
+        assert host.package("moby-engine").is_installed
```

Comment:
> And Moby, you can get stomped by Obie
You thirty six year old bald headed 🚬 , 💨  me
You don't know me, you're too old, let it go its over
Nobody listens to Techno